ide/css.lib/src/org/netbeans/modules/css/lib/Css3.g

```diff
@@ -36,6 +36,9 @@
 //    from the module directory (ide/css.lib)
 // 3. Rerun unittests
 // 4. Commit Css3.g together with generated Css3Lexer.java and Css3Parser.java
+//
```

Comment:
At the top of this file there's a line like this:
```
Author : Jim Idle, Temporal Wave LLC.
```
We may want to add an additional author there (be it Matthias or the NetBeans team or something). After all we don't want Jim Idle to be blamed by our sins... ;-)

Reply:
I'm inclined to agree, however I quote:

```
// This grammar is free to use providing you retain everyhting in this header comment
// section.
```

so I let it stand as is. Lines 19+ I think spell it out, that this is a modified version.